### DISCUSSION TITLE
OBOK don't convert to epub?

### DISCUSSION THREAD

**Discussion by AeliusSaionji** [noDRM/DeDRM_tools#517] [Q&A]
Is it possible to strip the drm of kepub without converting the output to epub? I'd like to keep the original format.

If not, I'd like to request that feature please :)

**Comment by bust4cap** [comment]
afaik its still a kobo compatible epub, just omitting the kepub part from the filename

**Reply by ElleKayEm** [reply]
This is true.  The file just needs to be renamed (filename.kepub.epub) for your Kobo to recognize it as such.

**Reply by AeliusSaionji** [reply]
> its still a kobo compatible epub

Kobo formally supports standard epub so of course it will be "compatible", but I want the extended feature set offered by kepub.

**Reply by ElleKayEm** [reply]
None of that is removed by Obok.  Just rename the file so your Kobo will use the correct renderer for it.

**Reply by AeliusSaionji** [reply]
ok, thanks

**Comment by JSWolf** [comment]
Click he ... at the eBook's listing on Kobo and you can download it as an ePub. Then you can use DeDRM to strip the DRM if there is any and have a retail ePub.

**Reply by ElleKayEm** [reply]
But they want kepub.  So they should stick to Obok.

**Comment by AeliusSaionji** [comment]
Finally got around to doing this.
There are two problems with the proposed answer.

1. Calibre seems to have no way to batch add formats, and no way to simply "rename" an extension of existing formats. Renaming the files outside of calibre causes calibre to consider those formats lost, and reimporting renamed files causes them to be "new books". You can only add new formats to existing books one at a time, which is tedious. The alternative of merging new imports with existing entries is also tedious.
2. Calibre thinks these epub-renamed-kepub files are drm locked and will not view them.

**Reply by ElleKayEm** [reply]
This plugin might help: https://www.mobileread.com/forums/showthread.php?t=211135

But I have no experience with it myself.
